src/expandableCalendar/__test__/index.spec.ts

```diff
@@ -0,0 +1,294 @@
+import {ExpandableCalendarProps, Positions} from '../index';
+import {toMarkingFormat, xdateToData} from '../../interface';
+import {ExpandableCalendarDriver} from '../driver';
+import {generateExpandableCalendarWithContext, testIdExpandableCalendar} from './expandableCalendarTestKit';
+//@ts-expect-error
+import {getMonthTitle} from '../../testUtils';
+import {UpdateSources} from '../commons';
+import times from 'lodash/times';
+import {NUMBER_OF_PAGES} from '../WeekCalendar';
+import {CalendarContextProviderProps} from 'react-native-calendars';
+
+const XDate = require('xdate');
+
+enum Direction {
+  LEFT = 'left',
+  RIGHT = 'right',
+}
+
+const testIdExpandable = 'myExpandableCalendar';
```

Comment:
There's another one in the testKit file, it's confusing...

Reply:
removed the duplicate